styles/pages/dashboard.module.css

```diff
@@ -0,0 +1,216 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  background-color: white;
+}
+
+.contentWrapper {
+  display: flex;
+  flex-grow: 1;
+  overflow: hidden;
+}
+
+.mainContent {
+  flex-grow: 1;
+  padding: 20px;
+  overflow-y: auto;
+}
+
+.contentColumns {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 2.5rem;
+}
+
+.recordSection {
+  margin-top: 5rem;
+}
+
+.recordSection, .uploadSection {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.recordSection .p {
+  margin-top: 1.5rem;
+  font-size: 1.15rem;
+  font-weight: bold;
+  color: black;
+}
+
+.recordButton {
+  width: 190px;
+  height: 190px;
+  background-color: #59DBBC;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  margin-bottom: 10px;
+}
+
+.divider {
+  width: 1px;
+  background-color: #ccc;
+  margin: 0 20px;
+}
+
+.uploadArea {
+  width: 20rem;
+  height: 20rem;
+  border: 2px dashed #ccc;
+  border-radius: 8px;
+  padding: 20px;
+  margin-bottom: 20px;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  background: #F8F8FF;
+}
+
+.uploadArea img {
+  margin-bottom: 10px;
+}
+
+.uploadButton {
+  background-color: #59DBBC;
+  color: white;
+  border: none;
+  padding: 10px 20px;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+.h2 {
+  color: black;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.7rem;
+}
+
+.p {
+  color: #666;
```

Comment:
Lets avoid hardcode colors

We need a way to have 5-6 colors palette in scss and then rehusthem

@text, @secondary @main @accent.  (THis one Im not sure how is done but i know we can)